Document that begin ... end use (#2147)
ocaml/ocaml.org: data/tutorials/language/0it_00_values_functions.md

- `42`, the result of `21 * 2`
- `713`, the result of `int_of_string "713"`

> When creating subexpressions, using a `begin ... end` syntax is also possible. This is the same as using brackets `( ... )`. As such, the above could also be rewritten and get the same result:

Reviewer: ```suggestion
When creating subexpressions, using `begin ... end` is also possible. This is the same as using brackets `( ... )`. As such, the above could also be rewritten and get the same result:
```

> When creating subexpressions, it is also possible to use a `begin ... end` syntax. These are equivalant to using a `( ... )` syntax. As such, the above could also be rewritten and get the same result:

Reviewer: ```suggestion
When creating subexpressions, using a `begin ... end` syntax is also possible. This is the same as using brackets `( ... )`. As such, the above could also be rewritten and get the same result:
```
